soda_machine.rb

```diff
@@ -7,12 +7,20 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+    @sodas.size
   end
 
   def find_soda(soda_brand)
+    @sodas.find{|soda| soda.brand == soda_brand}
   end
 
   def sell(soda_brand)
+    this_soda = find_soda(soda_brand)
+    if this_soda != nil
+      @sodas.delete_at(@sodas.index(this_soda))
```

Comment:
@amgando (@b0uma, @mashalifshin, @salogel42)

Done and [done](https://github.com/Devbootcamp/phase-1-checkpoint-challenge-2-soda-machine/pull/12)